tests/integration/components/paper/number-field-test.js

```diff
@@ -23,9 +23,16 @@ module('Integration | Component | {{paper/number-field}}', function(hooks) {
     assert.equal(this.get('number'), null);
   });
 
+  test('value gets formatted with two decimals', async function(assert) {
+    this.set('number', 42);
+    await render(hbs`{{paper/number-field value=number maxDecimals=2}}`);
```

Comment:
Where is this maxDecimals for in this test? For testing default decimals, I think you should remove every passed decimals configuration